SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Following Skosmos versions are currently being supported with security updates. The "current development branch" means the master branch of the repository, whereas the "maintenance branch" corresponds with the latest release of Skosmos.
```

Comment:
"currently being supported with security updates" - this doesn't say who is responsible. Could we say:

> currently being supported with security updates by the Skosmos development team at the National Library of Finland.

Minor grammar correction: "corresponds with" -> "corresponds to"

To be a bit more explicit about branches, could we express it like this:

> whereas the "maintenance branch" is a branch called `vX.X-maintenance`, where the version number X.X corresponds to the latest release of Skosmos.